adapter/internal/oasparser/model/types.go

```diff
@@ -265,6 +295,19 @@ func (apiProject *ProjectAPI) ProcessFilesInsideProject(fileContent []byte, file
 		}
 		apiProject.APIYaml = apiYaml
 		ExtractAPIInformation(apiProject, apiYaml)
+	} else if strings.Contains(fileName, rateLimitPoliciesFile) {
+		loggers.LoggerAPI.Debugf("fileName : %v available for the API project.", fileName)
+		rlPoliciesJsn, conversionErr := utills.ToJSON(fileContent)
+		if conversionErr != nil {
+			loggers.LoggerAPI.Errorf("Error occured rate limit policies file to json: %v", conversionErr.Error())
```

Comment:
shall we add error logs with codes?

Reply:
log codes not available in Choreo branch.